sdk/webpubsub/azure-messaging-webpubsubclient/azure/messaging/webpubsubclient/models/_models.py

```diff
@@ -450,37 +705,99 @@ def __init__(
 
 
 class OnServerDataMessageArgs:
-    def __init__(self, data_type: Union[WebPubSubDataType, str], data: Any, sequence_id: Optional[int] = None) -> None:
+    """Arguments for server data message
+
+    :ivar data_type: The data type of the message. Required.
+    :vartype data_type: WebPubSubDataType or str
```

Comment:
All of the complex object types in the docstrings for `paramtype`/`vartype`/etc need to be fully qualified or the reference doc linking wont work correctly.
```suggestion
    :vartype data_type: ~azure.messaging.webpubsubclient.WebPubSubDataType or str
```